src/components/form-slice/FormPartUpstream/FormSectionChecks.tsx

```diff
@@ -126,6 +128,7 @@ const FormItemChecksPassiveEnabled = () => {
 const FormSectionChecksPassiveCore = () => {
```

Comment:
<img width="464" alt="image" src="https://github.com/user-attachments/assets/7571c2f3-5990-4788-82c1-379707703b05" />

https://github.com/apache/apisix/blob/master/apisix/schema_def.lua#L289

Passive can only be enabled when Active is enabled

Reply:
Let's not consider this issue for this version.